src/main.js

```diff
@@ -2,8 +2,8 @@ import { createApp } from 'vue'
 import { createPinia } from 'pinia'
 
 import App from './App.vue'
-import router from './router'
-import '../node_modules/typeface-inter/inter.css'
+import router from './router' 
+import 'typeface-inter'
```

Comment:
Why are we loading font in js, instead of css?

Reply:
We are using "npm i typeface" to import google fonts to store locally. You can read more about typeface here https://www.npmjs.com/package/typefaces

Reply:
This package has not been updated since 2020.
Since CSS gives me an exclusive way to deal with fonts why should I use javascript to increase the overhead.
Also, we should not use too many npm packages as it affects the bundle size.

**Action Item**
Please download whatever font is used in the designs, import it using CSS and use it.